manifests/ssltools/nssdb.pp

```diff
@@ -0,0 +1,35 @@
+# Creates a nssdb
+class certs::ssltools::nssdb inherits certs {
```

Comment:
@ekohl: How do you suggest that we pass the parameters? That would mean we can't call the class with `include` but have to use `class {'':}` although we have to call the class several times?